crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -874,87 +877,91 @@ pub fn queue_mesh_view_bind_groups(
-            let depth_view = match prepass_textures.and_then(|x| x.depth.as_ref()) {
-                Some(texture) => &texture.default_view,
-                None => {
-                    &fallback_depths
-                        .image_for_samplecount(msaa.samples)
-                        .texture_view
-                }
-            };
-
-            let normal_view = match prepass_textures.and_then(|x| x.normal.as_ref()) {
-                Some(texture) => &texture.default_view,
-                None => {
-                    &fallback_images
-                        .image_for_samplecount(msaa.samples)
-                        .texture_view
-                }
-            };
-
             let layout = if msaa.samples > 1 {
                 &mesh_pipeline.view_layout_multisampled
             } else {
                 &mesh_pipeline.view_layout
             };
 
+            let mut entries = vec![
+                BindGroupEntry {
+                    binding: 0,
+                    resource: view_binding.clone(),
+                },
+                BindGroupEntry {
+                    binding: 1,
+                    resource: light_binding.clone(),
+                },
+                BindGroupEntry {
+                    binding: 2,
+                    resource: BindingResource::TextureView(
+                        &view_shadow_bindings.point_light_depth_texture_view,
+                    ),
+                },
+                BindGroupEntry {
+                    binding: 3,
+                    resource: BindingResource::Sampler(&shadow_pipeline.point_light_sampler),
+                },
+                BindGroupEntry {
+                    binding: 4,
+                    resource: BindingResource::TextureView(
+                        &view_shadow_bindings.directional_light_depth_texture_view,
+                    ),
+                },
+                BindGroupEntry {
+                    binding: 5,
+                    resource: BindingResource::Sampler(&shadow_pipeline.directional_light_sampler),
+                },
+                BindGroupEntry {
+                    binding: 6,
+                    resource: point_light_binding.clone(),
+                },
+                BindGroupEntry {
+                    binding: 7,
+                    resource: view_cluster_bindings.light_index_lists_binding().unwrap(),
+                },
+                BindGroupEntry {
+                    binding: 8,
+                    resource: view_cluster_bindings.offsets_and_counts_binding().unwrap(),
+                },
+                BindGroupEntry {
+                    binding: 9,
+                    resource: globals.clone(),
+                },
+            ];
+
+            // When using webgl with msaa, we can't create the fallback textures required by the prepass
+            // When msaa is disabled, we can't bind the textures either
```

Comment:
If I understood correctly...:
```suggestion
            // When using WebGL with MSAA, we can't create the fallback textures required by the prepass
            // When using WebGL, and MSAA is disabled, we can't bind the textures either
```